src/lib.rs

```diff
@@ -48,18 +48,18 @@ impl Logger {
 }
 
 struct StartTime;
-impl Key for StartTime { type Value = time::Instant; }
+impl Key for StartTime { type Value = time::Tm; }
```

Comment:
Are these changes to `StartTime` required for this PR? Otherwise I'd prefer to see them in another PR.